src/khoj/processor/embeddings.py

```diff
@@ -56,8 +56,8 @@ def embed_documents(self, docs):
                     try:
                         response.raise_for_status()
                     except requests.exceptions.HTTPError as e:
-                        print(f"Error: {e}")
-                        print(f"Response: {response.json()}")
+                        logger.error(f"Error: {e}")
```

Comment:
Can remove this change after I merged #628 .

Reply:
Removed that